sandbox-tools/merge-repos/src/config.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { IRepoSyncDetails } from "./types";
+
+/**
+ * This identifies both the initial source and the final destination for the merge.
+ * ie. The created PR will be created to merge back into this repo
+ */
+export const MERGE_ORIGIN_REPO = "opentelemetry/opentelemetry-sandbox-web-js";
+
+/**
+ * Identifies both the initial source and final destination branch for the merge
+ * ie. The created PR will be created to merger back into this branch for the Origin Repo
+ */
+export const MERGE_ORIGIN_MERGE_MAIN_BRANCH = "main";
+
+/**
+ * Identifies both the initial source and final destination branch for the merge
+ * ie. The created PR will be created to merger back into this branch for the Origin Repo
+ */
+export const MERGE_ORIGIN_STAGING_BRANCH = "auto-merge/repo-staging";
```

Comment:
This origin/destination branch will need to be created (from `main`) before the first run of the merge workflow and ideally before this PR is merged into `main` so there are less files to remove.

Once we agree that this should be the branch name we can create it.
This is the branch which is cloned locally (by the script) and has the master otel js and api-js repos merged into the `auto-merge` folder (as defined by const on line 50).

Reply:
I am fine with this branch name. Can we change it later on if we need to?

Reply:
I'm also fine with this branch name